### ISSUE TITLE
Orthodontist Appointment

### ISSUE THREAD

**Issue by OlenkaP** [RiLights/aligntime#39]
-  Change the name from 'Add Orthodontist Appointment' TO 'Orthodontist Appointment'
The rest as per screen mockups 

![image](https://user-images.githubusercontent.com/52235757/72333543-d88fb800-3720-11ea-938c-112db3a5bd1a.png)

**Comment by RiLights** [comment]
Are those parameters in the tab "More" or in the user menu?

**Comment by OlenkaP** [comment]
User menu, as we've decided to get rid of 'More' for now. I have just used the screen shot out of the app version i currently have

**Comment by RiLights** [comment]
Ok. Clear. Thanks!

**Comment by OlenkaP** [comment]
@RiLights in the app you had a question as to what is meant to be in this section...is this story helpful or would i need to update it?

**Comment by RiLights** [comment]
It definitely will be helpful! 
I would review general concept for Orthodontist Appointment as could be implemented in iOS original calendar app and we just sending and receiving data.

**Comment by OlenkaP** [comment]
@RiLights yes, absolutely would love to utilise IOS calendar, where possible.
Would would we require in order to be able to do that

**Comment by RiLights** [comment]
That was just a general concept. Right now I even don't know what we have to do to implement that
